.github/workflows/testNginxForAzureDeployV1.yml

```diff
@@ -0,0 +1,55 @@
+# File: .github/workflows/testNginxForAzureDeployV1.yml
+
+name: Test Github action to update NGINX for Azure configurations and certificates
+on:
+  - push
+  - workflow_dispatch
+
+env:
+  NGINX_DEPLOYMENT_NAME: '0707221656'
+  NGINX_TRANSFORMED_CONFIG_DIR_PATH: /etc/nginx/
+  NGINX_ROOT_CONFIG_FILE: nginx.conf
+  TEST_RESOURCE_GROUP_NAME: george-test
+  NGINX_RESOURCE_LOCATION: westcentralus
+  NGINX_CERT_DETAILS: '[{"nginx_cert_name": "server4", "keyvault_secret": "https://isolated-kv.vault.azure.net/secrets/server", "certificate_virtual_path": "/etc/ssl/certs/server4.crt", "key_virtual_path": "/etc/ssl/certs/server4.key"  }, {"nginx_cert_name": "server5", "keyvault_secret": "https://isolated-kv.vault.azure.net/secrets/acr-auth", "certificate_virtual_path": "/etc/ssl/certs/server5.crt", "key_virtual_path": "/etc/ssl/certs/server5.key"  }]'
+
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  Update-NGINX-Configuration:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout repository'
+        uses: actions/checkout@v2
+      - name: 'AZ CLI Login'
+        uses: azure/login@v1
+        with:
+            client-id: ${{ secrets.AZURE_CLIENT_ID }}
+            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+            subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      - name: 'Sync NGINX certs to NGINX on Azure instance'
+        uses: george-ngugi/nginx-for-azure-deploy-action@v1
```

Comment:
Instances of this need to be changed to `nginxinc/nginx-for-azure-deploy-action`